content/en/blog/_posts/2025-XX-XX-endpoints-deprecation.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: blog
+title: "Endpoints formally deprecated in favor of EndpointSlices"
+slug: endpoints-deprecation
+date: 2025-XX-XX
+draft: true
+author: >
+  Dan Winship (Red Hat)
+---
+
+Since the addition of [EndpointSlices] several years ago, the
+Endpoints API in Kubernetes has been gathering dust. New Service
+features like [dual-stack networking] and [topology-aware routing] are
+only supported via the EndpointSlice API, so all service proxies,
+Gateway API implementations, and similar controllers have had to be
+ported from using Endpoints to using EndpointSlices. At this point,
+the Endpoints API is really only there to avoid breaking end user
+workloads and scripts that still make use of it.
+
+We are now beginning the process of officially deprecating the
```

Comment:
Avoid “We“ when there is a single author. See https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we and also the provisos in https://kubernetes.io/docs/contribute/review/reviewing-prs/#blog